core/arch/arm/kernel/spmc_sp_handler.c

```diff
@@ -213,10 +213,32 @@ void spmc_sp_msg_handler(struct thread_smc_args *args,
 			cpu_spin_unlock(&caller_sp->spinlock);
 			caller_sp = NULL;
 			break;
+#ifdef ARM64
```

Comment:
This assumes that if OP-TEE Core is 64-bit so is the SP.

Reply:
Currently we don't support 32 bit partitions yet. I would like to add this later in a different patch series.

Reply:
Currently we don't support 32 bit partitions yet. I would like to add this later in a different patch series.

Reply:
OK